mani_skill/envs/tasks/tabletop/place_sphere.py

```diff
@@ -62,25 +62,23 @@ def _default_sim_config(self):
 
     @property
     def _default_sensor_configs(self):
-        pose = sapien_utils.look_at(eye=[0.3, 0, 0.2], target=[-0.1, 0, 0])
```

Comment:
why is the original camera pose changed?

Reply:
It's just to align with other tabletop tasks' camera poses like PushCube/PickCube/StackCube's default poses (they use the same poses)